### DISCUSSION TITLE
모바일 방송에서 TTS 기능을 쓰려면 어떻게 해야하나요?

### DISCUSSION THREAD

**Discussion by summercatkr** [Lastorder-DC/chatreader-kor#46] [문의] [answered]
안녕하세요.
프리즘라이브로 스마트폰 모바일 방송을 하며,
TTS 기능이 없는 채팅창 위젯을 사용하고 있습니다.

개발자님 채팅봇을 모바일 방송에 적용시키고 싶은데, 
어떻게 해야할까요?

**Comment by Lastorder-DC** [comment]
#47 모바일 사용을 고려하지 않고 만든것이라... 가능할지 살펴보겠습니다.

**Comment by Lastorder-DC** [comment, marked as answer]
모바일에서도 사용하기 위해서는 모바일 크롬을 사용하시거나 아마존 폴리를 연동해두어야 합니다.
이는 대다수 방송프로그램에서 웹페이지를 띄우는데 사용하는 크로미움은 크롬과 달리 구글 TTS를 무료 사용할수 없기 때문입니다.
물론 구글 TTS를 유료로 연동해두면 이론상 사용이 가능하긴 합니다만 현재 후원금 규모(놀랍게도 0원입니다)로는 절대 불가능한 일입니다.

기능 구현후 PR 남겨주시는분이 계시다면 검토후 적용해보겠으나 이외 적극적으로 해당 기능 추가하진 않겠습니다.
